web-ui/src/components/feedback_request_card/feedback_request_subcard/FeedbackRequestSubcard.jsx

```diff
@@ -58,6 +80,22 @@ const FeedbackRequestSubcard = (props) => {
     handleReminderNotification();
   }
 
+  const Submitted = (props) => {
+      if (props.dueDate) {
+        let today = new Date();
+        let due = new Date(props.dueDate[0], props.dueDate[1] - 1, props.dueDate[2]);
+        if ((!props.submitDate || props.submitDate === undefined) && today > due) {
```

Comment:
The check `!props.submitDate || props.submitDate === undefined` is logically equivalent to `!props.submitDate`